docs/installation/certificate.asciidoc

```diff
@@ -0,0 +1,355 @@
+// to display images directly on GitHub
+ifdef::env-github[]
+:encoding: UTF-8
+:lang: en
+:doctype: book
+:toc: left
+:imagesdir: ../images
+endif::[]
+
+////
+
+    This file is part of the PacketFence project.
+
+    See PacketFence_Installation_Guide.asciidoc
+    for authors, copyright and license information.
+
+////
+
+
+= PacketFence Certificates
+
+== Summary
```

Comment:
Isn't that auto-generated? @nqb can tell

Reply:
We can keep it, it's added in all .asciidoc files even if it's not really used.